pkg/apis/alicloud/v1alpha1/types_infrastructure.go

```diff
@@ -95,6 +95,14 @@ type Zone struct {
 	Worker string `json:"worker"`
 	// Workers specifies the worker CIDR to use.
 	Workers string `json:"workers"`
+	// NatGatewayConfig specifies the eips to bind
+	// +optional
+	NatGateway *NatGatewayConfig `json:"natGateway,omitempty"`
+}
+
+// NatGatewayConfig contains specified eips to bind nat gateway
+type NatGatewayConfig struct {
+	IPAddresses []string `json:"ipAddresses,omitempty"`
```

Comment:
```suggestion
  // <doc string here, please>
  // +optional
	IPAddresses []string `json:"ipAddresses,omitempty"`
```

+ see above comment wrt `[]string` vs `string`.